src/components/Home/LeftPanel.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component, PropTypes } from 'react';
+import { Link } from 'react-router';
+import picture from '../../assets/user-picture.png'
+import Signup from '../../containers/Signup'
+import '../../styles/LeftPanel.scss';
+
+class LeftPanel extends Component {
+  render() {
+    const { showEditProfile, toggleEditProfile, user } = this.props;
+
+    return (
+      !showEditProfile ?
```

Comment:
This is difficult to read.
Try to refactor like this:
Define two const before return statment:
```
const editView = (
  <div className="container">
    <h1 className="title">target</h1>
    <div className="circle box"><img className="img-center" src={picture}/></div>
    <h3>{user.name}</h3>
    <span><span className="link-to-edit" onClick={toggleEditProfile}>Edit</span> / Profile</span>
    <div className="line-separator small-line margin-top-target"></div>
  </div>
);
const signUpView = (
  <div>
    <Signup isEdit={showEditProfile}/>
    <span className="link-to-edit" onClick={toggleEditProfile}>Show Profile</span>
  </div>
)
```

Then in the return:
```
return !showEditProfile ? editView : signUpView
```